### PR TITLE
Feat: Add aws credentials

### LINKED ISSUE #47: Add support for AWS Credentials resource/data source

### PR DESCRIPTION
**Pull request by roni-frantchi**
Resolves #47

* Resource
* Data
* UT
* Integration

### CODE REVIEW

File: env0/resource_aws_credentials.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+			},
+			"arn": {
+				Type:        schema.TypeString,
+				Description: "the aws role arn",
+				Required:    true,
+			},
+			"external_id": {
+				Type:        schema.TypeString,
+				Description: "the aws role external id",
+				Required:    true,
```
Comment (yaronya) [inline env0/resource_aws_credentials.go:35]: @roni-frantchi you can make this one `Sensitive: true` as well 💡

Comment (roni-frantchi) [reply env0/resource_aws_credentials.go:37]: # 💡

Comment (roni-frantchi) [reply env0/resource_aws_credentials.go:37]: https://github.com/env0/terraform-provider-env0/pull/48/commits/c9fd26e9ed6ba64dc67527d9ba566353f3da9f84

**Review by HeverFarber — APPROVED** [review]

**Comment by roni-frantchi** [comment]
[TF Provider - Cloud credentials resources types](https://trello.com/c/Jaer1RFZ/3290-tf-provider-cloud-credentials-resources-types)

**Pull request merged**